docs/platform-validation/index.md

```diff
@@ -0,0 +1,141 @@
+
+
+### Data accuracy check
+
+For Intel X86 platforms, since the RAPL based system power collection is available on BareMetal, Kepler uses [Power Ratio Modeling](https://sustainable-computing.io/design/power_model/).
+
+Such power attribution mechanism accuracy should be checked.
+
+We need to introduce an independent and platform agnostic way to collect the node components power info.
+
+Due to the root priviledge limit on the access of the RAPL SYSFS files, we need to use priviledged container to perform the test.
+
+
+For node level power accuracy check, the comparison logic is simple and strightforward, while for pod/container level power accuracy check, the logic is a little bit complicated and need some assumptions.
+
+(1) RAPL energy is node/package level, so we could only use RAPL sampling delta to calculate the power change introduced by application deployment and undeployment.
+
+(2) On the other hand, application power consumption data could be queried from Prometheus where Kepler's exported time-series metrics are aggregated. 
+
+(3) On a clean/idle platform, without other tenants' workloads interference, we could have a simple assumption that the power increase introduced by the specific application deployment is equal to the test container calculated power delta in (1).
+
+(4) On a busy platform, with other tenants' workloads running in parallel, in container or in VM, for instance, the scenarios become complicated. Since the interference power may be fluctuated and unable to detect, the Kepler `Power Ratio Modeling` check criteria is unknown, now we just dump all the available data and leave the result for evaluation after platform validation test. 
+
+(5) On some CPU/GPU-intensive workloads deployment platform, such as AI/AIGC inference pipeline's worker node, the `validator` raw data could be used to measure the approximate power consumption contribution of those inferencing pods/containers. We can then compare the Kepler demonstrated power consumption data, either in PromQL query result(automation) or on Grafana Dashboard(manual test). 
+
+This is also meaningful test case for the carbon footprint accuracy check on AI/AIGC workloads. 
+
+## Validation workflow
+
+See example Github action workflow [here](https://github.com/sustainable-computing-io/kepler/blob/main/.github/workflows/platform-validation.yml).
+
+The workflow is similar as the e2e integration test workflow, has two phases jobs:
+
+### Artifacts build.
+
+Build both the `Kepler` container image and the `Validator` container image.
+
+The current Validator image refers to the Kepler image build mechanism to build from a GPU image to support GPU case. We may simply build it from UBI image later.
+
+Since it is an independent energy collector other than Kepler, those eBPF stuffs will not be bumped into the container.
+
+### Platform validation test.
```

Comment:
TODO: please get a script for these steps.

Reply:
Currently these steps have been automated in Github action workflow, see Kepler PR#902.
.gitub/workflows/platform-validation.yml and hack/verify.sh.